overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -10069,7 +10069,7 @@
+          "desc:8": "Craft any 6 pieces of Witches\u0027 Altars! Place them in a 2x3 formation and place any piece of Carpet on top to form the multiblock.\n\nThis structure provides power for all your Bewitchment machines to function. Witches\u0027 Cauldrons, Distilleries, Spinning Wheels, Braziers and Circle Rites wouldn\u0027t function without sufficient Altar power in a 16 block radius. To start running Bewitchment machines obtaining around a thousand or two Altar power would be a good idea.\n\nTo boost the power stored by your Altar, you need to spice up the immediate area with a bunch of wildlife! Trees, crops, growables, grass, flowers, fungi, anything you can think of. And variety helps! Two different types of crops will boost the power of your Altar more than two of the same type.\n\nTo boost the power further, you can also place any of the following items on the top 6 slots (in increasing order of effectiveness):\n\n- Cups: Flower Pot / Glass Bottle, Goblet, Flying Ointment, or Filled Goblet\n\n- Vanilla Swords or Athame (with the Diamond Sword and the Athame being the best).\n\n- Wands or lighting: Torch / Redstone Torch, Candles, Blaze Rod, Candelabra, End Rod, Leonard\u0027s Wand, Caduceus.\n\n- Pentacles: Head / Skeleton Skull, Redstone / Nether Quartz / Lapis Lazuli / Opal, Wither Skeleton Skull, Amethyst / Diamond / Emerald / Garnet / Prismarine Shard, Grimoire Magia, Idols, Pentacle, Demon Heart, any Statue, Dragon Head.\n\nMore information on the Bewitchment Wiki:\nhttps://bewitchment.fandom.com/wiki/Witches%27_Altar/1.12\n"
```

Comment:
inaccurate. bewitchment machines need very little capacity, only the cauldron and rituals need significant amounts